### PR TITLE
Add support for unicode escape sequences per spec

### PR DESCRIPTION
**Pull request by rw-access**
## Issues
EQL escape sequences for unicode `\u{...}` weren't working as expected.

## Details
Added support for `\u{ ... }` escapes, per the [spec](https://www.elastic.co/guide/en/elasticsearch/reference/current/eql-syntax.html#eql-syntax-escape-characters)

### CODE REVIEW

File: eql/__init__.py [hunk 1]
```diff
@@ -66,7 +66,7 @@
     Walker,
 )
 
-__version__ = '0.9.10'
+__version__ = "0.9.11"
```
Comment (brokensound77) [inline eql/__init__.py:69]: This broke the version regex parsing from setup.py

Comment (rw-access) [reply eql/__init__.py:69]: oh yeah
```suggestion
__version__ = '0.9.11'
```

File: tests/test_parser.py [hunk 2]
```diff
@@ -511,6 +511,19 @@ def test_term_extraction(self):
         simple_extracted = extract_query_terms(network_event + "| unique process_name, user_name\n\n| tail 10")
         self.assertListEqual(simple_extracted, [network_event.strip()])
 
+    def test_unicode_escape(self):
```
Comment (brokensound77) [inline tests/test_parser.py:514]: can you add another test with rendered unicode such as `☢`

Comment (brokensound77) [reply tests/test_parser.py:514]: Even though that should have always worked, apart from this update

Comment (rw-access) [reply tests/test_parser.py:514]: what do you mean by "rendered"? `String().render()`?

**Review by brokensound77 — APPROVED** [review]
LGTM, thanks! `☢☢☢`

File: tests/test_parser.py (hunk 2 above)
Comment (brokensound77) [reply tests/test_parser.py:514]:
```sql
process where process.name == "☢"
```

no biggie if not

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # Event Query Language - Changelog
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+# Version 0.9.11
+_Released 2021-09-23_
```
Comment (brokensound77) [inline CHANGELOG.md:5]:
```suggestion
_Released 2021-09-22_
```
